src/linter/root.go

```diff
@@ -627,11 +627,7 @@ func (d *RootWalker) enterPropertyList(pl *stmt.PropertyList) bool {
 
 		nm := p.Variable.Name
 
-		typ, errText := d.parsePHPDocVar(p.PhpDocComment)
-		if errText != "" {
-			d.Report(p.Variable, LevelWarning, "phpdocLint", errText)
-		}
-
+		typ := d.parsePHPDocVar(p.PhpDocComment)
```

Comment:
Why remove this check?

Reply:
It wasn't checking anything, parsePHPDocVar always returns an empty string as an error text.